cypress/integration/item/chatbox/chatbox.spec.js

```diff
@@ -48,7 +48,8 @@ describe('Chatbox Scenarios', () => {
     );
     cy.get(`#${CHATBOX_ID} #${CHATBOX_INPUT_BOX_ID} button`).click();
     cy.wait('@postItemChatMessage').then(({ request: { body } }) => {
-      expect(body.body).to.equal(message);
+      expect(body.body.message).to.equal(message);
```

Comment:
Should this one also be deeply compared?

Reply:
@dialexo In this case, the `message` property is just a string, so i don't think that it needs to be deeply compared.